addons/fleet/models/fleet_vehicle.py

```diff
@@ -330,7 +331,7 @@ class FleetVehicleOdometer(models.Model):
     value = fields.Float('Odometer Value', group_operator="max")
     vehicle_id = fields.Many2one('fleet.vehicle', 'Vehicle', required=True)
     unit = fields.Selection(related='vehicle_id.odometer_unit', string="Unit", readonly=True)
-    driver_id = fields.Many2one(related="vehicle_id.driver_id", string="Driver", readonly=False)
+    driver_id = fields.Many2one('res.partner', string="Driver", compute='_compute_driver_id', store=True)
```

Comment:
Why not a stored-readonly-related field ? 

On the other hand, why storing this information on all the odometers ? Seems overkill right ?

Reply:
Mmmh. I think I get it, it's to avoid loosing the info about who encoded the odometer in case we change the driver of the vehicle right ?

Then we should improve the compute method to look into driver_logs (fleet.vehicle.assignation.log) (to be a little smarter than storing the same information everywhere).

Reply:
Btw storing this field would have implied a migration script. ;)

Reply:
> Mmmh. I think I get it, it's to avoid loosing the info about who encoded the odometer in case we change the driver of the vehicle right ?

Yes that's it I wanted to store it in case the driver of the car changes.

> Then we should improve the compute method to look into driver_logs (fleet.vehicle.assignation.log) (to be a little smarter than storing the same information everywhere).

Okay I will look at this and update the branch :)

Reply:
> Btw storing this field would have implied a migration script. ;)

Yes I was aware of it but I waited the review stage to see if we let it as it is or not.